MvvmCross.Forms/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Xamarin.Forms;
+
+[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Bindings))]
+[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Converters))]
+[assembly: XmlnsDefinition("http://mvvmcross.com/bind", nameof(MvvmCross.Forms.Views))]
+namespace MvvmCross.Forms
+{
+    internal class AssemblyInfo
```

Comment:
This won't work unless you also tell msbuild not to generate its own default AssemblyInfo file. Can we add these assembly things elsewhere?

Reply:
It's not about adding the file. I could add those attributes on any file basically. We just need to make sure it is not getting linked away.

Reply:
Yeah, but it doesn't build. Because the SDK style projects generate AssemblyInfo, now you've duplicated it.